src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
```

Comment:
Consider adding error handling for the case where the '.logo' element is not found. This will prevent potential runtime errors if the element does not exist in the DOM.